docs/src/components/home/sections/HeroSection.tsx

```diff
@@ -102,43 +102,37 @@ export const HeroSection = () => {
             </Grid>
           ) : null}
 
-          {platform === 'flutter' ? (
-            <TerminalCommand
-              command={`flutter pub add ${frameworkInstallScript}`}
-              variant="hero"
-            />
-          ) : (
-            <TerminalCommand
-              command={`npm install ${frameworkInstallScript}`}
-              variant="hero"
-            />
+          {platform === 'swift' ? null : (
+            <TerminalCommand command={frameworkInstallScript} variant="hero" />
           )}
 
-          <Flex direction="row">
-            <Button
-              size="large"
-              variation="primary"
-              className="docs-home-cta"
-              as="a"
-              // @ts-ignore // IGNORE
-              href={`/${platform}/getting-started/installation`}
-            >
-              Get started building
-              <Icon ariaLabel="" as={MdChevronRight} fontSize="xl" />
-            </Button>
-            {platform === 'react' && showEditor ? (
+          {platform === 'swift' || platform === 'android' ? null : (
```

Comment:
@eeatonaws @atierian @reesscot we don't have a `Getting Started` page for android or swift and not sure we'll be able to write one by monday so I just removed those sections on the intro page for now

Reply:
I had added a getting started page for Android (page with setup instructions is [here](https://github.com/aws-amplify/amplify-ui/pull/3677/files#diff-fa9b2fad4185c329f46132efd49a17adb89ce8cd7640c701c510fea1fb0f27ba)).